aws-android-sdk-cognitoidentityprovider/src/main/java/com/amazonaws/mobileconnectors/cognitoidentityprovider/CognitoUser.java

```diff
@@ -2332,6 +2334,28 @@ private void deleteAttributesInternal(final List<String> attributeNamesToDelete,
         cognitoIdentityProviderClient.deleteUserAttributes(deleteUserAttributesRequest);
     }
 
+    public void revokeTokens() {
```

Comment:
on iOS PR, we decided to expose 2 APIs on the Cognito User. Please see this comment https://github.com/aws-amplify/aws-sdk-ios/pull/3638#discussion_r669191279  for more details as to the discussion that lead to this decision. I'm not sure if we need to align the APIs to this level of detail 
1. `isSessionRevocable` which returns bool for checking origin_jti claim 
2. `revokeToken` which is 1:1 with the LLC API and has a type that is from the codegenerated response types `AWSCognitoIdentityProviderRevokeTokenResponse`

Reply:
> 1. `isSessionRevocable` which returns bool for checking origin_jti claim

The revokeToken returns either null or RevokeTokenResult. If revoke token fails due to origin_jti claim not found or server side error/exception, we just log the error and swallow the exception.

> 2. `revokeToken` which is 1:1 with the LLC API and has a type that is from the codegenerated response types `AWSCognitoIdentityProviderRevokeTokenResponse`

Added code changes.